.github/workflows/ci.yml

```diff
@@ -1,71 +1,19 @@
 name: CI
-on: [push, pull_request]
-env:
-  RAILS_ENV: test
-  DATABASE_URL: postgresql://postgres:@localhost/test
-  DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: true
-  BUNDLE_WITHOUT: "journald:development:console:mysql2:sqlite"
+
+on: pull_request
```

Comment:
Btw, it's also fine to keep building on push (like we used to). That way you get a shared cache for future PRs. You can consider limiting it to just master and any stable branches.

Reply:
Let's add on push to at least mater branch, wouldn't hurt I guess.